content/en/blog/falcosidekick-openfaas.md

```diff
@@ -0,0 +1,571 @@
+---
+title: Falcosidekick + OpenFaas = a Kubernetes Response Engine, Part 2
+date: 2021-04-11
+author: Batuhan Apaydın
+slug: falcosidekick-openfaas
+---
+
+We recently talked about a concept called _"Kubernetes Response Engine"_, and we achieved this by using `Falco`
+
++ `Falcosidekick` + `Kubeless`. But as you might guess, _Falcosidekick_ project is evolving day after day, which means
+  new output are added to the project. With the
+  release [`2.22.0`](https://github.com/falcosecurity/falcosidekick/releases/tag/2.22.0), we are proud to
+  support [`OpenFaaS`](https://www.openfaas.com) as a new output for _Falcosidekick_. We can achieve the same concept
+  called _"Kubernetes Response Engine"_ but this time by using _"OpenFaaS"_ instead of _"Kubeless"_.
```

Comment:
```suggestion
We recently talked about a concept called _"Kubernetes Response Engine"_, and we achieved this by using `Falco` + `Falcosidekick` + `Kubeless`. But as you might guess, `Falcosidekick` project is evolving day after day, which means new outputs are added.
With the release [`2.22.0`](https://github.com/falcosecurity/falcosidekick/releases/tag/2.22.0), we are proud to
  support [`OpenFaaS`](https://www.openfaas.com) as a new output for _Falcosidekick_. This allows us to achieve the same concept, _"Kubernetes Response Engine"_, but this time by using _"OpenFaaS"_ instead of _"Kubeless"_.
```

Reply:
done